scripts/run-in-cluster.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Constants
+
+PROJECT_NAME="business-index-exporter"
+
+# Variables
+
+scriptsDirectory=$(dirname $0)
+libsDirectory=${scriptsDirectory}/../lib
+
+libraries=$(ls -m ${libsDirectory}/*.jar | grep -v ${PROJECT_NAME} | tr -d ' \n')
+
+# Job submission
+
+spark-submit --master yarn-cluster --jars ${libraries} ${libsDirectory}/*${PROJECT_NAME}*.jar $@
```

Comment:
I do not like the fact that the `spark-submit` `--jars` option requires a list as opposed to a directory. This is a nice workaround 👍